optional-dependencies.txt

```diff
@@ -4,3 +4,5 @@ matplotlib
 pysrt
 SpeechRecognition
 pySaliencyMap
+PIL
```

Comment:
I think PIL is superseded by pillow, which is already a hard dependency, so maybe see if you can use the Image class in pillow instead.